PowerShell Contributing.md

```diff
@@ -0,0 +1,479 @@
+
+When in doubt, just share what you encountered to check if it's a bug in PowerShell or not.
+Good places to share are:
+
+- [Twitter](https://www.twitter.com), tag your tweet with #pshelp
+- The [PowerShell Discord channel](https://aka.ms/psdiscord)\*
+- The [PowerShell Slack channel](https://aka.ms/psslack)\*
+- The [PowerShell IRC channel](https://poshcode.org/#irc)\*
+- The [PowerShell/PowerShell Gitter channel](https://gitter.im/PowerShell/PowerShell)
+
+\*) Discord, Slack and IRC are bridged via a channel called #Bridge. Read more about this on <poshcode.org>.
+
+### 2. Help confirm bugs
+
+> Requires
+> 
+> - GitHub account
+
+Confirming bugs is going through recent bug report issues and trying to reproduce the bug.
+This helps assess if a reported bug is really a bug or not.
+
+A lot of new bug reports are going to have the ['Issue-Discussion'](https://github.com/PowerShell/PowerShell/issues?q=is%3Aissue+is%3Aopen+label%3AIssue-Discussion) label, instead of 'Issue-Bug'. 
+This is because when you first open an issue on the PowerShell project, the automatic label for a bug report is Issue-Discussion, which typically won't be changed until a maintainer or collaborator has seen that the bug has been confirmed / reproduced by others or can reproduce it themselves.
+
+You can comment in the issue whether or not you could repro it, and on which version of PowerShell.
+Always try to reproduce a bug on the latest version of PowerShell.
+Make sure you read the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md) and the [known issues](https://docs.microsoft.com/powershell/scripting/whats-new/).
+
+If a bug is reproducible, it can be tested and fixed.
+If a bug is not reproducible, more research may be needed.
+You may be able to help the user who filed the bug with that.
+
+> Tip: Consider 'Watching' the github repo.
+You'll get a feel for what is going on daily.
+
+### 3. Comment on language change proposals
+
+> Requires
+> 
+> - GitHub account
+
+The following things currently require an RFC:
+
+- Large-scale changes to cmdlets
+- **Any** changes to core language features
+- New language features
+- Deprecation of outdated features
+
+Anyone can submit a proposal and  request comments from the community and the PowerShell Team.
+Typically, an RFC will remain open for two months, to give everyone the opportunity to comment.
+At this point, the PowerShell Committee will make the decision to approve or reject the RFC.
+
+The RFC may also be discussed in the monthly call if there is reason to.
+
+The full process is explained here: <https://github.com/PowerShell/powershell-rfc/blob/master/RFC0000-RFC-Process.md>
+
+By commenting on an RFC, you can influence how an idea is going to be implemented, if at all.
+Comments are valuable for the community and the committee to make a balanced decision, weighing all voices.
+
+### 4. Attend the monthly PowerShell community call
```

Comment:
I would start with explaining what this is and then mention the requirements. Most important content first.